ui/droppable.js

```diff
@@ -413,6 +405,62 @@ $.ui.ddmanager = {
 	}
 };
 
+// DEPRECATED TODO: switch return back to widget declaration at top of file when this is removed
+if ( $.uiBackCompat !== false ) {
+
+	// DialogClass option
```

Comment:
This shouldn't be uppercased. To get around the comment rule, "Backcompat for dialogClass option" or something like that.